package.json

```diff
@@ -24,7 +24,7 @@
     "coverage": "node $(npm bin)/istanbul cover -x \"**/third_party/**\" _mocha -- $(find */test -name '*-test.js') --timeout 10000 --reporter progress",
     "coveralls": "npm run coverage && cat ./coverage/lcov.info | coveralls",
     "start": "node ./lighthouse-cli/index.js",
```

Comment:
I'm leaning towards not incurring that cost every start, but a note in the README or something would be nice :)